src/modules/audio/audio.c

```diff
@@ -122,22 +117,20 @@ static void onPlayback(ma_device* device, void* output, const void* _, uint32_t
 static void onCapture(ma_device* device, void* output, const void* input, uint32_t frames) {
   // note: ma_pcm_rb is lockless
   void *store;
+  size_t bytesPerFrame = SampleFormatBytesPerFrame(CAPTURE_CHANNELS, OUTPUT_FORMAT);
```

Comment:
A little confused about why this is using `OUTPUT_FORMAT`, especially if the capture format can be changed.

Reply:
Yeah that's a bug. Fixed. Well spotted!